sample/src/main/java/vehicles/AbstractVehicle.java

```diff
@@ -0,0 +1,40 @@
+package vehicles;
+
+import people.Person;
+
+import java.time.LocalDate;
+
+/**
+ * An abstract class for implementing different kinds of {@link Vehicle}.
+ *
+ * @author http://twitter.com/manoelcampos[Manoel Campos da Silva Filho]
+ */
+public class AbstractVehicle implements Vehicle {
+    private int manufacturingYear;
```

Comment:
I completely agree.  I guess I'm just looking to demonstrate different aspect of the doclet.